chpl/chpl-auth/src/main/java/gov/healthit/chpl/auth/EmailOverrider.java

```diff
@@ -121,6 +121,6 @@ private String getEmailDomain(final String someEmail) {
     }
 
     private String getForwardToEmail() {
-        return env.getProperty("emailBuilder_config_forwardAddress");
+        return env.getProperty("emailBuilder.config.forwardAddress");
```

Comment:
The "_" was used because on the servers we set these values in the `setenv.sh`.  I found that environment variables with "." do not work in the *nix environments.
https://unix.stackexchange.com/questions/93532/exporting-a-variable-with-dot-in-it

Reply:
Well drat. I was hoping there wasn't a good reason for this...